src/puppetlabs/puppetdb/query.clj

```diff
@@ -65,7 +65,7 @@
             [puppetlabs.puppetdb.jdbc :as jdbc]
             [puppetlabs.puppetdb.time :refer [to-timestamp]]
             [puppetlabs.kitchensink.core :refer [parse-number keyset valset order-by-expr?]]
-            [puppetlabs.puppetdb.scf.storage-utils :refer [db-serialize sql-as-numeric sql-array-query-string sql-regexp-match sql-regexp-array-match]]
+            [puppetlabs.puppetdb.scf.storage-utils :as sutils :refer [db-serialize sql-as-numeric sql-array-query-string sql-regexp-match sql-regexp-array-match]]
```

Comment:
Would it be possible to convert all these refers to just using the namespace? This is slowly what I've been doing as the refer's become too long, the advantage of using namespace all the time is that if we follow a rough convention for short names, we can just pick up functions & move them around to other files without having to do as much work in the require block.